client/src/components/Cell/CellMain.tsx

```diff
@@ -84,21 +108,26 @@ const CellMain = ({
           {t('rankUp')}
         </FantasyButton>
       )}
+
       {owner && (brute.xp < xpNeededForNextLevel ? fightsLeft > 0 ? (
         <Stack spacing={1} sx={{ alignItems: 'center', mt: 1 }}>
           <Text bold sx={{ pl: 1 }}>{t('callToFight')}</Text>
-          <Link to={`/${brute.name}/arena`}>
```

Comment:
THe arrows don't belong here, but on the sides of the screen, so this UI change needs to be reverted